Automattic/Gravatar-SDK-Android
# [Tooling] Automation for downloading translations

### Description

This PR adds basic Fastlane automation for downloading translation strings from GlotPress.

Additionally, it also sets up Danger in the project, mainly to do Ruby-related checks for now, but of course the checks can be expanded as needed.

### Testing Steps

1. Checkout this branch
1. Update one of the translation strings on GlotPress ([ui module](https://translate.wordpress.com/projects/gravatar/gravatar-android-sdk/gravatar-ui/), [quick editor module](https://translate.wordpress.com/projects/gravatar/gravatar-android-sdk/gravatar-quick-editor/))
1. Run `bundle install` + `bundle exec fastlane download_translations`
1. If there are updates, the tool should download the updated strings of the corresponding locale, update and commit them in the project
